### PR TITLE
7.0 Add class CarrierAccount, so the specifics carriers modules use all the same base

### PR DESCRIPTION
**Pull request by florian-dacosta**
I had to close my last pull request because I did not target the right branch.
So here is my new pull request.
You can see the closed one for more informations : 
https://github.com/OCA/carrier-delivery/pull/1

### CODE REVIEW

File: base_delivery_carrier_label/delivery.py [hunk 1]
```diff
@@ -96,3 +96,41 @@ def __get_carrier_type_selection(self, cr, uid, context=None):
             'delivery.carrier.option',
             'carrier_id', 'Option'),
     }
+
+
+class CarrierAccount(orm.Model):
```
Comment (bwrsandman) [inline base_delivery_carrier_label/delivery.py]: Can you put this class/model in its own python named `carrier_account.py`?

Comment (florian-dacosta) [inline base_delivery_carrier_label/delivery.py]: Done!
Thanks for your comment

Comment (bwrsandman) [inline base_delivery_carrier_label/delivery.py]: Thank you, that is much better!

**Comment by rvalyi** [comment]
:+1: LGTM

**Comment by yvaucher** [comment]
:+1: 
Thanks for your work

To answer @guewen question in https://github.com/OCA/carrier-delivery/pull/1
When you deal with a carrier to generate specific labels, you can need an account to generate part of your label, in other case you can get an image as a stamp to add to your label.

It's true you won't need this model if you install only those modules for using default label.

But this seems generic enough to me to be included in base of labels. This is used by postlogistics and by chronopost.

**Pull request merged**